backend/api_core/serializers.py

```diff
@@ -13,3 +13,26 @@ class Meta:
 
     def get_content(self, item):
         return json.loads(item.content)
+
+
+class HospitalSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Hospital
+        fields = ['name', ]
+
+
+class ComorbiditySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Comorbidity
+        fields = ['name', ]
+
+
+class CaseSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Case
+        initial_hospital = HospitalSerializer(read_only=True, many=False)
+        final_hospital = HospitalSerializer(read_only=True, many=False)
+        comorbidities = ComorbiditySerializer(read_only=True, many=True)
+        fields = ['case_number', 'collision_index', 'date_created', 'date_modified', 'gender', 'age', 'county',
```

Comment:
Source should have its own serializer, since it is a model of its own. Please expose the url field from the source foreign key when writing the extra source serializer.